docs/design/tdd_composition_workloads.md

```
@@ -97,7 +97,7 @@ SyncedGroup (1kHz)
 | Overrun handling              | Logged per-child                  | Interrupt must return fast (no overrun logs)|
 | Tick sync                     | True sync                         | Sync possible but looser (main may lag)     |
 
-### Thread Model (MCU: ISR + main loop)
+### Thread Model_v1 (MCU: ISR + main loop)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)